sql/2021/security/https_server_redirects.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# Prevalence of server redirects from HTTP to HTTPS over Jan 2021 to Aug 2021
```

Comment:
Why Jan to August?

Either way we should limit it to July as that's the max data set we're supposed to all be using.